platform-controller/src/test/java/org/hobbit/controller/ExperimentTimeoutTest.java

```diff
@@ -49,11 +49,12 @@
 
     @Before
     public void init() {
+        // set max execution time to 1s
+        System.setProperty("MAX_EXECUTION_TIME", "1000");
```

Comment:
It might be better to use the environmental variable setting method used in other JUnit tests since it is more realistic than setting a system property (from my point of view).

You need to define a rule as done at
https://github.com/hobbit-project/core/blob/master/src/test/java/org/hobbit/core/components/BenchmarkControllerTest.java#L69-L70

After that you can use it to define env variables (that normally can not be set by the java program)
https://github.com/hobbit-project/core/blob/master/src/test/java/org/hobbit/core/components/BenchmarkControllerTest.java#L85-L92